content/blueprints/gcp.md

```diff
@@ -0,0 +1,1277 @@
+      type: text|file
+      data: Private key in PEM format if text, otherwise path to a file with private key
+
+    *default:* {}
+  * `name`
+    Optional SSL certificate name. By default it will be SSL certificate id.
+
+    *type:* string
+    *default:*
+  * `certificate`
+    Certificate (self-signed or obtained from CA) in PEM format. Expected format is:
+      type: text|file
+      data: Certificate in PEM format if text, otherwise path to a file with certificate
+
+    *default:* {}
+  * `gcp_config`
+    A dictionary of values to pass to authenticate with the GCP API.
+
+    *default:* {}
+  * `additional_settings`
+    Additional setting for target proxy
+
+    *default:* {}
+  * `use_external_resource`
+    Indicate whether the resource exists and use existing (true) or if Cloudify should create new resource (false).
+
+    *type:* boolean
+    *default:* False
+
+
+
+
+
+## cloudify.gcp.nodes.StaticIP
+**Derived From:** cloudify.gcp.nodes.GlobalAddress
+
+Alias for GlobalAddress for backward compatibility.
+
+
+
+**Properties:**
+
+
+
+
+
+
+
+## cloudify.gcp.nodes.SubNetwork
+**Derived From:** [cloudify.nodes.Subnet]({{< relref "blueprints/built-in-types.md" >}})
+
+A GCP Subnetwork. Must be connected to a Network using `cloudify.gcp.relationships.contained_in_network`.
+
+Only networks with the `auto_subnets` property disabled can be used.
+
+
+
+**Properties:**
+
+
+  * `subnet`
+    The subnet, denoted in CIDR form (i.e. '10.8.0.0/20') Subnets must be unique and non-overlapping within a project. See https://cloud.google.com/compute/docs/subnetworks#networks_and_subnetworks
+
+    *type:* string
+    *default:*
+  * `region`
+    The region this subnet is in. See https://cloud.google.com/compute/docs/regions-zones/regions-zones
+
+    *type:* string
+    *default:*
+  * `use_external_resource`
+    Indicate whether the resource exists or if Cloudify should create the resource.
+
+    *type:* boolean
+    *default:* False
+  * `name`
+    Optional SubNetwork name. The instance ID will be used by default.
+
+    *default:*
+  * `gcp_config`
+    A dictionary of values to pass to authenticate with the Google Cloud Platform API.
+
+    *default:* {}
+
+
+
+### Example
+
+{{< gsHighlight  yaml  >}}
+
+my_net:
+  type: cloudify.gcp.nodes.Network
+  properties:
+    auto_subnets: false
+
+my_subnet:
+  type: cloudify.gcp.nodes.SubNetwork
+  properties:
+    subnet: 10.8.0.0/20
+  relationships:
+    - type: cloudify.gcp.relationships.contained_in_network
+      target: my_net
+
+my_instance:
+  type: cloudify.gcp.nodes.Instance
+  properties:
+    ...
+  relationships:
+    - type: cloudify.gcp.relationships.contained_in_network
+      target: my_subnet
+
+{{< /gsHighlight >}}
+
+If you want to use an exsisting SubNetwork (`use_external_resource: true`) then you must supply the `name` and `region` properties. This is because SubNetwork names are not unique across the whole project, only within a region.
```

Comment:
~~If you want to~~ **To** use an exsisting SubNetwork (`use_external_resource: true`) ~~then~~ **,** you must supply the `name` and `region` properties. This is because SubNetwork names are not unique across the whole project, only within a region.